packages/ricos-types/web/src/plugin-add-buttons.ts

```diff
@@ -31,7 +31,7 @@ export interface PluginAddButton {
 
   equals: (button: PluginAddButton) => boolean;
 
-  toToolbarItemConfig: () => IToolbarItemConfigTiptap;
+  toToolbarItemConfig: (disabled?: boolean) => IToolbarItemConfigTiptap;
```

Comment:
To align with the text-button API change the function signature
```suggestion
  toToolbarItemConfig: (options?: { disabled: boolean }) => IToolbarItemConfigTiptap;
```